### ISSUE TITLE
Zigbee2MQTT on docker bootloop

### ISSUE THREAD

**Issue by bbacid7** [Koenkk/zigbee2mqtt#12070]
### What happened?

Hi there,

I am trying to connect zigbee2mqtt to mqtt to make zigbee devices available in Homebridge. Both are installed via Docker on my Synology NAS (DSM 7). I am using the SONOFF Zigbee USB Dongle Plus 3.0 as the zigbee coordinator.

Homebridge is up and running on docker (exposing works to Homekit)

I succesfully flashed the stick ([CC1352P2_CC2652P_launchpad_coordinator_20220219.zip](https://github.com/Koenkk/Z-Stack-firmware/blob/master/coordinator/Z-Stack_3.x.0/bin/CC1352P2_CC2652P_launchpad_coordinator_20220219.zip))

I succesfully set up mqtt (version 2.0.14 openssl) on docker

The dongle was detected by Homeassistant and I was also able to add a zigbee device

However I wasnt able to run zigbee2mqtt in docker. It is bootlooping and makes the whole NAS crash.

The configuration.yaml file is set up correctly as far as I can tell.

configuration.yaml:

> Home Assistant integration (MQTT discovery)
> homeassistant: true
> 
> allow new devices to join
> permit_join: true
> MQTT settings
> mqtt:
> MQTT base topic for zigbee2mqtt MQTT messages
> base_topic: zigbee2mqtt
> MQTT server URL
> server: 'mqtt://192.XXX.X.XXX:1883'
> MQTT server authentication, uncomment if required:
> user: my_user
> password: my_password
> 
> Serial settings
> serial:
> Location of CC2652P USB sniffer
>   port: /dev/ttyUSB0
> 

The mqtt config file also seems fine:

> persistence true
persistence_location /mosquitto/data
log_dest file /mosquitto/log/mosquitto.log
#password_file /mosquitto/config/pwfile
allow_anonymous true
listener 1883 192.XXX.X.XXX

The port for the coordinator is correctly identified and listed as found with the ls command via ssh access (through Putty) to the NAS root.

The log of the zigbee2mqtt on docker before it crashes is the following:

> 2022-04-03 18:34:33: Logging to console and directory: '/app/data/log/2022-04-03.18-34-33' filename: log.txt
2022-04-03 18:34:33: Starting Zigbee2MQTT version 1.25.0 (commit #6f1460e)
2022-04-03 18:34:33: Starting zigbee-herdsman (0.14.20)
2022-04-03 18:34:54: Error while starting zigbee-herdsman
2022-04-03 18:34:54: Failed to start zigbee
2022-04-03 18:34:54: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
2022-04-03 18:34:54: Exiting...
2022-04-03 18:34:54: Error: SRSP - SYS - getExtAddr after 6000ms
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
Using '/app/data' as data directory
2022-04-03 18:34:59: Logging to console and directory: '/app/data/log/2022-04-03.18-34-58' filename: log.txt
2022-04-03 18:34:59: Starting Zigbee2MQTT version 1.25.0 (commit #6f1460e)
2022-04-03 18:34:59: Starting zigbee-herdsman (0.14.20)
2022-04-03 18:35:19: Error while starting zigbee-herdsman
2022-04-03 18:35:19: Failed to start zigbee
2022-04-03 18:35:19: Exiting...
2022-04-03 18:35:19: Error: Failed to connect to the adapter (Error: SRSP - SYS - ping after 6000ms)
    at ZStackAdapter.start (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/zStackAdapter.ts:102:27)
    at Controller.start (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:123:29)
    at Zigbee.start (/app/lib/zigbee.ts:58:27)
    at Controller.start (/app/lib/controller.ts:100:27)
    at start (/app/index.js:101:5)
Using '/app/data' as data directory
2022-04-03 18:35:24: Logging to console and directory: '/app/data/log/2022-04-03.18-35-23' filename: log.txt
2022-04-03 18:35:24: Starting Zigbee2MQTT version 1.25.0 (commit #6f1460e)
2022-04-03 18:35:24: Starting zigbee-herdsman (0.14.20)
2022-04-03 18:35:30: Not connected to MQTT server!
2022-04-03 18:35:30: Cannot send message: topic: 'zigbee2mqtt/bridge/state', payload: 'offline
2022-04-03 18:35:30: Disconnecting from MQTT server
TypeError: Cannot read properties of undefined (reading 'end')
    at MQTT.disconnect (/app/lib/mqtt.ts:111:21)
    at Controller.stop (/app/lib/controller.ts:196:9)
    at stop (/app/index.js:105:5)
    at process.handleQuit (/app/index.js:111:9)
Search

I don't understand why it fails to start zigbee, maybe because of the "Failed to connect to the adapter" log? But then I also wouldn't understand why it fails to connect to the adapter when it works in Homeassistant. Also confusing to me why it  fails to start and then manages to start.

And the "Not connected to MQTT server!" log is also a mystery to me since the setup of the config files seems to be correct.

This is my first foray into all of this so I may have done something wrong, are there any pointers in these logs or config files towards whereI may have gone wrong?

Basically all I wanted was to add 1-2 IKEA tradfri motion sensors to Homekit without using the IKEA gateway and the SONOFF coordinator was the cheapest solution I could find. I feel I am too deep in now to jump ship and want to get this working.

Hopefully someone can help. Thanks in advance!

### Zigbee2MQTT version

1.25.0

### Adapter firmware version

20220219

### Adapter

SONOFF Zigbee 3.0 USB Dongle Plus

**Comment by bbacid7** [comment]
SOLVED! The problem was the connection of ZHA to the dongle. After I disabled the connection in ZHA it worked flawlessly.
